src/explore-education-statistics-admin/src/pages/release/data/components/ApiDataSetPreviewTokenCreateForm.tsx

```diff
@@ -0,0 +1,87 @@
+import Button from '@common/components/Button';
+import ButtonGroup from '@common/components/ButtonGroup';
+import ButtonText from '@common/components/ButtonText';
+import {
+  Form,
+  FormFieldCheckbox,
+  FormFieldset,
+  FormFieldTextInput,
+} from '@common/components/form';
+import FormProvider from '@common/components/form/FormProvider';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import Yup from '@common/validation/yup';
+import React, { useMemo } from 'react';
+import { ObjectSchema } from 'yup';
+
+interface FormValues {
+  label: string;
+  terms: boolean;
+}
+
+interface Props {
+  onCancel: () => void;
+  onSubmit: (label: string) => void;
+}
+
+export default function ApiDataSetPreviewTokenCreateForm({
+  onCancel,
+  onSubmit,
+}: Props) {
+  const validationSchema = useMemo<ObjectSchema<FormValues>>(() => {
+    return Yup.object({
+      label: Yup.string().required('Enter a label'),
+      terms: Yup.boolean()
+        .required('The terms of usage must be accepted')
+        .oneOf([true], 'The terms of usage must be accepted'),
```

Comment:
Minor, but should this be 'The terms of usage must be agreed' to be consistent with what the checkbox actually says?